charts/testkube-api/values.yaml

```diff
@@ -177,6 +177,12 @@ affinity: {}
 ## Specify priorityClassName for Testkube API Deployment
 priorityClassName: ""
 
+## Specify hostNetwork for Testkube API Deployment
+hostNetwork: ""
```

Comment:
Could you please duplicate the values into `testkube/values.yaml` file under `testkube-api` section and run `helm-docs` to update README.md?

Reply:
sure thing, done.